.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/google/snappy.git
 [submodule "rocksdb_sys/rocksdb"]
 	path = librocksdb-sys/rocksdb
-	url = https://github.com/facebook/rocksdb.git
+	url = https://github.com/hayesgm/rocksdb.git
```

Comment:
I think it's a bad idea to use a fork rather than official repository.

Reply:
Definitely. This is meant as a placeholder until that patch is merged into mainline in rocksdb.

Reply:
https://github.com/facebook/rocksdb/commit/ee4bd4780b321ddb5f92a0f4eb956f2a2ebd60dc

Reply:
Pushed a commit to use proper FB version again